### PR TITLE
cm show infor

### PR DESCRIPTION
**Pull request by ChuThiThuyhang**

### CODE REVIEW

**Comment by ChuThiThuyhang** [comment]
pull about show information of plans, tours, rates, provinces.

**Comment by ChuThiThuyhang** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -21,8 +21,12 @@
 Route::get('/login', 'LoginController@login');
 Route::post('/login', 'LoginController@postLogin');
 
-Route::group(['middleware' => 'checkAdminLogin', 'prefix' => 'admincp', 'namespace' => 'Admin'], function() {
+Route::group(['middleware' => 'checkAdminLogin', 'prefix' => 'admincp', 'namespace' => 'Admin'], function(){
```
Comment (BunnyPi04) [inline routes/web.php]: convention

**Comment by BunnyPi04** [comment]
commented

File: database/migrations/2018_07_25_082311_create_users_table.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            $table->string('phonenumber');
+            $table->string('username');
+            $table->string('password');
+            $table->string('address');
+            $table->integer('role');
+            $table->rememberToken();
+            $table->timestamps();
+        });
+    }
+    public function down()
```
Comment (BunnyPi04) [inline database/migrations/2018_07_25_082311_create_users_table.php]: function cách 1 dòng trắng nhé

**Comment by BunnyPi04** [comment]
commented

**Pull request merged**